<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>236</width>
    <height>99</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="ListView_B">
     <property name="text">
      <string>ListView</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="TableView_B">
     <property name="text">
      <string>TableView</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="TreeView_B">
     <property name="text">
      <string>TreeView</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
